components/inputs/test/input-textarea.visual-diff.js

```diff
@@ -9,36 +9,64 @@ describe('d2l-input-textarea', () => {
 
 	before(async() => {
 		browser = await puppeteer.launch();
-		page = await visualDiff.createPage(browser, { viewport: { width: 800, height: 1300 } });
+		page = await visualDiff.createPage(browser, { viewport: { width: 800, height: 2000 } });
 		await page.goto(`${visualDiff.getBaseUrl()}/components/inputs/test/input-textarea.visual-diff.html`, { waitUntil: ['networkidle0', 'load'] });
 		await page.bringToFront();
 	});
 
 	after(async() => await browser.close());
 
-	['wc', 'sass'].forEach((type) => {
```

Comment:
I added a few more visual-diff tests for things specific to `d2l-input-textarea`.  There's a little duplication between this and the Sass, but there are enough differences that combining them starts to get messy, and I'd like to keep them both separate in code, and separate in reporting.